lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
Although, since it is a module perhaps `module_function` would be more appropriate: 
https://github.com/bbatsov/ruby-style-guide#module-function